features/step_definitions/factory_steps.rb

```diff
@@ -17,6 +16,35 @@
   end
 end
 
+Given /^I accept a proposed organisation called "(.*?)" with email "(.*?)"$/ do |name, email|
+  steps %{
+    Given the following proposed organisations exist:
+      | name    | description         | address        | postcode | telephone | website            | email    | donation_info        | non_profit |
+      | #{name} | Mourning loved ones | 30 pinner road | HA5 4HZ  | 520800000 | http://#{name}.org | #{email} | www.pleasedonate.com | true       |
+    When I accept the proposed_organisation named "#{name}"
+  }
+end
+
+Then /^"(.*?)" is (notified|invited|ignored) as an organisation admin of "(.*?)"$/ do |email, action, name|
+  case action
+  when 'notified'
+    steps %{
```

Comment:
I used the extensively before, but as the nesting got bigger and wading through them became more and more of a chore (the jump to thing not working, but also stack trace not very revealing) I did a lot of research and seemed to find blog posts telling me to go pure ruby in my cucumber steps.  We refactored and I liked it.  Can't find that original blog post, but I think there was a trend in the community to avoid nested steps: 

http://blog.mattwynne.net/2012/04/26/a-vision-for-cucumber-2-0/

Since moving away from them I haven't looked back.  Ensuring that every cucumber step consists of direct capybara calls definitely seemed very appealing. Although I see there are some work arounds suggested here:

https://makandracards.com/makandra/18905-how-to-not-repeat-yourself-in-cucumber-scenarios